packages/Benefit/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@tds/core-benefit",
+  "version": "1.0.0-dev",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-core.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-core/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": "^16.8.0",
```

Comment:
Let's use 16.8.2. https://github.com/telus/tds-core/pull/999#discussion_r268784882

Reply:
I changed it to 16.8.2